config/locales/server.en.yml

```diff
@@ -612,6 +612,7 @@ en:
         one: "%{count} user has been added to the group."
         other: "%{count} users have been added to the group."
     errors:
+      default_notification_level_users_limit: "Default notification level can't be set on groups larger than %{count} users. Incorrect group: %{group_name}"
```

Comment:
We should have `one:` and `other:` for translations with count to cover the different singular and plural form of `user`

Reply:
Yes I hesitated, my position is that a limit of 1/0 seems totally stupid, so this extra string seems useless 🤷‍♂️